resources/address_format/CH.json

```diff
@@ -7,7 +7,9 @@
 		"locality",
 		"postalCode"
 	],
-	"uppercase_fields": [],
```

Comment:
I don't know, could be, will that be a problem? Do you want me to revert that file?